### ISSUE TITLE
Run cargo update?

### ISSUE THREAD

**Issue by ximon18** [NLnetLabs/krill#447]
Quite a few dependencies are outdated. Should we review their release notes and bump them up prior to the next release?

```bash
$ git log -1
commit 23be0b5ef7bd2ce4472315e5db4b866e76360ea5 (HEAD -> v0.8.1-bis, origin/v0.8.1-bis)
...

$ cargo outdated -R
warning: Feature rt-core of package tokio has been obsolete in version 1.4.0
Name           Project  Compat   Latest   Kind    Platform
----           -------  ------   ------   ----    --------
bcder          0.5.0    0.5.1    0.6.0    Normal  ---
bytes          0.5.6    ---      1.0.1    Normal  ---
clokwerk       0.3.3    0.3.4    0.3.4    Normal  ---
fern           0.5.9    ---      0.6.0    Normal  ---
futures        0.3.7    0.3.13   0.3.13   Normal  ---
futures-util   0.3.7    0.3.13   0.3.13   Normal  ---
hex            0.4.2    0.4.3    0.4.3    Normal  ---
hyper          0.13.8   0.13.10  0.14.4   Normal  ---
libc           0.2.86   0.2.91   0.2.91   Normal  cfg(unix)
libflate       1.0.3    1.0.4    1.0.4    Normal  ---
log            0.4.11   0.4.14   0.4.14   Normal  ---
openssl        0.10.30  0.10.33  0.10.33  Normal  ---
oso            0.8.2    ---      0.11.1   Normal  ---
regex          1.4.2    1.4.5    1.4.5    Normal  ---
reqwest        0.10.9   0.9.24   0.11.2   Normal  ---
reqwest        0.9.24   ---      0.11.2   Normal  ---
rustc_version  0.2.3    ---      0.3.3    Build   ---
serde          1.0.117  1.0.125  1.0.125  Normal  ---
serde_json     1.0.59   1.0.64   1.0.64   Normal  ---
syslog         4.0.1    ---      5.0.0    Normal  cfg(unix)
tokio          0.2.22   0.2.25   1.4.0    Normal  ---
tokio-rustls   0.14.1   ---      0.22.0   Normal  ---
toml           0.5.7    0.5.8    0.5.8    Normal  ---
uuid           0.8.1    0.8.2    0.8.2    Normal  ---
```

**Comment by timbru** [comment]
Yes we should. We normally do this whenever we do a release - but it's fine to track this as an issue.

**Comment by ximon18** [comment]
FYI final releases of the `oauth2` and `openidconnect` crates are imminent. They shouldn't require any code changes or contain any functional differences, it's just that the beta release 'soak' period will be over and the releases will be promoted to final versions:

- https://github.com/ramosbugs/oauth2-rs/issues/99
- https://github.com/ramosbugs/openidconnect-rs/issues/41

**Comment by ximon18** [comment]
See also #366.

**Comment by ximon18** [comment]
There's also an Oso release planned for tomorrow that fixes https://github.com/osohq/oso/issues/788 which Krill multi-user policy files currently have a small workaround for, and thus could be removed by upgrading to that tomorrow.

**Comment by timbru** [comment]
Ok, let's get the oso release in the linked PR then.

**Comment by ximon18** [comment]
Closing as updates have since been done and osohq/oso#788 was fixed in Oso v0.12.0 and we are currently using v0.12.4.
